src/viewer.js

```diff
@@ -3362,6 +3362,7 @@ class VolumeImageViewer {
           studyInstanceUID: metadata.StudyInstanceUID,
           seriesInstanceUID: metadata.SeriesInstanceUID,
           sopInstanceUIDs: [metadata.SOPInstanceUID],
+          referencedSeriesSequence: metadata.ReferencedSeriesSequence[0],
```

Comment:
Sequences should be array ```referencedSeriesSequence: metadata.ReferencedSeriesSequence```. If you are using only the series instance uid then you can ```referencedSeriesInstanceUID: metadata.ReferencedSeriesSequence[0].SeriesInstanceUID```